frontend/src/App.js

```diff
@@ -1,22 +1,17 @@
-import logo from './logo.svg';
 import './App.css';
+import TopBar from './Components/TopBar';
+import GameEnv from './Components/GameEnv';
+import BottomBar from './Components/BottomBar';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+    <div className='App'>
+      <header className='App-header'>
+        <div style={{ width: '80%' }}>
+          <TopBar userName='Reeve' role='Civilian' />
```

Comment:
Will the game screen components be extracted into a game screen page component later?